packages/button/src/ButtonContent/ButtonContent.styles.ts

```diff
@@ -70,3 +70,37 @@ export const buttonContentSizeStyle: Record<Size, string> = {
     gap: 8px;
   `,
 };
+
+export const textlessLoadingStyles = (
+  loadingContentRef: React.MutableRefObject<HTMLDivElement | null>,
+) => css`
+  width: ${loadingContentRef.current?.style.width};
```

Comment:
I don't see this showing up in the browser, do you need this? Wouldn't the width inherit the child width if there is button text?

Reply:
Good call! Thank you.